amundsen_application/static/js/components/common/EditableText/index.tsx

```diff
@@ -136,9 +127,17 @@ class EditableText extends React.Component<EditableTextProps, EditableTextState>
                   </div>
                 </Tooltip>
               </Overlay>
-            </div>
           }
-          <div className='editable-text'>{ this.state.value }</div>
+          <div className={"editable-text " + (this.state.value ? "" : "no-value")}>
+            { this.state.value }
+            <a className={ "edit-link " + (this.state.isDisabled ? "hidden" : "") }
```

Comment:
For what reason do we want to have the edit target rendered here? It looks like the edit target will end up getting rendered regardless of whether or not the component is editable.

nit: Visually, does the anchor tag happen to take on some default `visited` style as well? (which i think would be undesirable)

Reply:
The current idea is to put the edit link inline with the description and hide/show on hover using opacity so it remains in the DOM flow. When editing is disabled, the `hidden` class will use `display: none` and remove this element from DOM flow. Alternatively we could just not render this link altogether and there might be a very minor performance improvement.

As for `:visited` I believe this uses browser history and the `href` attribute to determine if an anchor is visited or not. Since the href is `JavaScript:void(0)` I believe we can ignore this case.